scale/prometheus.go

```diff
@@ -0,0 +1,46 @@
+package scale
+
+import (
+	"errors"
+	"net/http"
+
+	dto "github.com/prometheus/client_model/go"
+	"github.com/prometheus/common/expfmt"
+)
+
+func getMetrics(url string) (map[string]*dto.MetricFamily, error) {
```

Comment:
In Prometheus terminology, we may name it `pullMetrics` to indicate that there is an HTTP request for them.